ui-v2/app/controllers/dc/services/show.js

```diff
@@ -19,7 +19,12 @@ export default Controller.extend(WithEventSource, WithSearching, {
     // This method is called immediately after `Route::setupController`, and done here rather than there
     // as this is a variable used purely for view level things, if the view was different we might not
     // need this variable
-    set(this, 'selectedTab', 'instances');
+
+    // set(this, 'selectedTab', 'instances');
+    // FIXME: Just to make it easier to build for the moment
+    // We'll also need to use a similar or the same approach as our
+    // didAppear thing see components/code-editor.js plus others
+    set(this, 'selectedTab', 'routing');
```

Comment:
Yeah definitely this was pretty much the first thing we addressed in https://github.com/hashicorp/consul/pull/6939.

We now do this with a new `isInViewport` method on our `dom` service (powered by IntersectionObserver). We'll probably move everywhere else we need to do similar to this method. The `isInViewport` method will also probably be refined slightly in the future as multiple components should be able to use the same `IntersectionObserver`. What we have there is fine for now as we only use it on one component that is essentially a singleton component.

We chose not to make these tabs linkable, so unless that changes in the future I'm thinking of being able to set that tab you want focussed by default whilst developing using the Web Inspector/Cookies like we do with out other dev settings (similar to what you do with your mirage logging/faker salt setting). I've realized since fixing this that during development, that extra click is a bit too annoying 😆

Reply:
This also means that test here won't pass until https://github.com/hashicorp/consul/pull/6939 is merged down onto here.